service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/util/SectionUtil.java

```diff
@@ -0,0 +1,68 @@
+package uk.nhs.adaptors.oneoneone.cda.report.util;
+
+import lombok.experimental.UtilityClass;
+import uk.nhs.connect.iucds.cda.ucr.POCDMT000002UK01ClinicalDocument1;
+import uk.nhs.connect.iucds.cda.ucr.POCDMT000002UK01Component2;
+import uk.nhs.connect.iucds.cda.ucr.POCDMT000002UK01Component5;
+import uk.nhs.connect.iucds.cda.ucr.POCDMT000002UK01Entry;
+import uk.nhs.connect.iucds.cda.ucr.POCDMT000002UK01Section;
+import uk.nhs.connect.iucds.cda.ucr.POCDMT000002UK01StructuredBody;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+@UtilityClass
+public class SectionUtil {
```

Comment:
This class is a copy paste, we don't need the entire code.

Reply:
Module removed...